src/components/MultipleChoiceQuestion.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import Immutable from 'immutable';
+
+import Dropdown from 'components/Dropdown';
+
+export default class MultipleChoiceQuestion extends PureComponent {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    question: PropTypes.instanceOf(Immutable.Map).isRequired,
+    nextQuestionOptions: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const { id, question, nextQuestionOptions } = this.props;
+    const defaultNextQuestionId = question.get('default_next_question_id');
```

Comment:
That's what I had before, but due to the way the database schemas are split up (there is a schema for non-conditional questions and conditional questions - they aren't treated as the same), having the model be structured like this seemed more intuitive when @dinahshi and I discussed it. See https://docs.google.com/document/d/11XTedlqNgk_ETftkewYyul_oRKDei0e4bmOxvWowJd8/edit for more details.